cmd/ebitenmobile/_files/EbitenView.java

```diff
@@ -348,6 +390,11 @@ private static int getAxisMask(InputDevice joystickDevice) {
         if (naxes >= 6) {
             axisMask |= ((1 << SDL_CONTROLLER_AXIS_TRIGGERLEFT) | (1 << SDL_CONTROLLER_AXIS_TRIGGERRIGHT));
         }
+        // Also add an indicator bit for whether the sorting order has changed.
+        // This serves to disable outdated gamecontrollerdb.txt mappings.
+        if (haveZ && havePastZBeforeRZ) {
+            axisMask |= 0x8000;
```

Comment:
This is also a magic number in the SDL source, right?

Reply:
Yes; it only exists in the Java code actually, as it's only used for looking up SDL game controller DB mappings, and C code doesn't use it in any way.

It might get a name in the future, but I doubt it.